database.yml

```diff
@@ -42,10 +42,12 @@ test:
   username:
   password:
 
-# production:
-#   adapter: postgresql
-#   encoding: unicode
-#   database: test-postgres_production
-#   pool: 5
-#   username:
-#   password:
+production:
```

Comment:
We can actually move `database.yml` to `config/database.yml` and `sinatra-activerecord` will pick it up: https://github.com/janko-m/sinatra-activerecord#setup